apps/web/src/pages/changes/components/ChangesTableLayout.tsx

```diff
@@ -138,7 +138,7 @@ export const ChangesTable = ({
     <Table
       data-test-id={dataTestId}
       loading={loading}
-      data={changes || []}
+      data={changes ? changes.slice().reverse() : []}
```

Comment:
its supposed to be an api leve change,
you need to sort in `/changes` api(while querying the DB)
https://github.com/novuhq/novu/blob/next/apps/api/src/app/change/changes.controller.ts#L53